app/tut/ga/reamhra.md

```diff
@@ -1,9 +1,93 @@
 ---
-title: Réamhra Teagaisc
+title: Réamhrá Teagaisc
 ---
 
 # Fáilte!
 
-**Fáilte go dtí Teagasc Setanta**.
+**_Fáilte go dtí an Teagasc Setanta_**.
 
-Tá súil againn go bhfuil tú réidh chun foghlaim gach rud faoi ríomhchlárúchán, agus conas a oibríonn é agus an Ghaeilge le chéile.
+Tá súil againn go bhfuil tú réidh chun foghlaim gach rud faoi [[ríomhchlárúchán|programming]], agus
+conas a oibríonn é agus an Ghaeilge le chéile.
+
+Níl ach cúpla rudaí le míniú sula mbeidh tú réidh le tosú.
+
+## Aistriúcháin
+
+Má fheiceann tú focal le folíne mar [[seo|D'aimsigh tú mé!]], is focal é le
+[[aistriúchán|translation]].  Cuir do luch thar an focal nó cliceáil air chun a aistriúchán a
+fheiceáil. [[Bain triail as!|Try it out!]]
+
+## Eagarthóirí Setanta
+
+Ar fud an [[teagasc|tutorial]] seo, aimseoidh tú [[eagarthóirí|editors]] Setanta mar seo:
+
+{{{
+>-- Is eagarthóir Setanta é seo
+scríobh("Dia duit!")
+}}}
+
+Is féidir linn eagarthóirí a úsáid chun ríomhchláir Setanta a scríobh. Clóscríobhaimid ríomhchlár
+isteach sa bosca ar chlé agus brúimid <iron-icon class="play" icon="av:play-arrow"></iron-icon> chun
+é a tosú. Beidh torthaí an ríomhchláir le feiceáil ar dheis.
+
+Brúigh an cnaipe <iron-icon class="play" icon="av:play-arrow"></iron-icon> chun an cód san
+eagarthóir a rith.  Bá chóir go bhfuil "Dia duit!" scríofa ar an taobh dheis.
+
+## An Consól
+
+Ar an taobh dheis, ba chóir go bhfuil seo le feiceáil:
+
+> Dia Duit!
+
+Is é seo an consól. Is féidir linn an consól a úsáid chun [[teachtaireachtaí|messages]] a léamh agus
+a scríobh.  Sa ríomhchlár roimhe (`scríobh("Dia duit!")`{.setanta}), d'úsáideamar an
+[[gníomh|action]] `scríobh`{.setanta} chun "Dia duit!" a scríobh amach ar an gconsól.
+
+Freisin, is féidir linn an consól a úsáid chun teachtaireachtaí a sheoladh chuig an ríomhchlár. Bain
+triail as an gcéad ríomhchlár eile seo a rith; Ba chóir go scríobhfadh sé "Cad is ainm duit?" sa
```

Comment:
>  Bain triail as an gcéad ríomhchlár eile seo a rith

Arís measaim go bhfuil an abairt seo clunky. Táim ag déanamh iarracht "try running this next program" a rá